core/templates/services/UpgradedServices.ts

```diff
@@ -705,6 +709,19 @@ export class UpgradedServices {
     upgradedServices['EndExplorationValidationService'] =
       new EndExplorationValidationService(
         upgradedServices['baseInteractionValidationService']);
+    upgradedServices['ExplorationCreationBackendApiService'] =
```

Comment:
Do you need to add it here? If not, lets not add it

Reply:
Removed it